packages/devtools_app/lib/src/network/flutter/network_screen.dart

```diff
@@ -272,3 +193,128 @@ class NetworkScreenBodyState extends State<NetworkScreenBody> {
     );
   }
 }
+
+class HttpRequestsTable extends StatelessWidget {
+  const HttpRequestsTable({
+    Key key,
+    @required this.networkController,
+    @required this.requests,
+  }) : super(key: key);
+
+  static MethodColumn methodColumn = MethodColumn();
+  static UriColumn uriColumn = UriColumn();
+  static StatusColumn statusColumn = StatusColumn();
+  static DurationColumn durationColumn = DurationColumn();
+  static TimestampColumn timestampColumn = TimestampColumn();
+
+  final NetworkController networkController;
+  final List<HttpRequestData> requests;
+
+  @override
+  Widget build(BuildContext context) {
+    return OutlineDecoration(
+      child: FlatTable<HttpRequestData>(
+        columns: [
+          methodColumn,
+          uriColumn,
+          statusColumn,
+          durationColumn,
+          timestampColumn,
+        ],
+        data: requests,
+        keyFactory: (HttpRequestData data) => ValueKey<HttpRequestData>(data),
+        onItemSelected: (item) {
+          networkController.selectHttpRequest(item);
+        },
+        autoScrollContent: true,
```

Comment:
just want to verify that this is what we want here. We should be scrolling to the bottom?

Reply:
Yup! I think so - we want to make sure that the user is aware of new network events as they occur. If they do manually scroll the table away from the end, the table will stop auto-scrolling.